app/views/divisions/division_not_found.html.haml

```diff
@@ -0,0 +1,6 @@
+- content_for :title, 'Division not found'
+%h1= yield :title
+
+No Division found. If you entered a date, please make
```

Comment:
The other option is to redirect to a generic 404 page, which includes helpful information for people to either go to the home page, search or correct the url. It looks like we don't have a 404 page at all at the moment!! Yikes.

Because this error is probably quite rare to hit, it's probably not worth the effort/maintenance of a specific 404. What do you recon?